pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -36,6 +37,7 @@ type ConsoleServerCLIConfigBuilder struct {
 	statusPageID      string
 	customProductName string
 	customLogoFile    string
+	routerCA          string
```

Comment:
This property should prob be `CAFile`, as it may be the router-CA, or it may not.